nle/tests/test_envs.py

```diff
@@ -119,6 +119,37 @@ def test_default_wizard_mode(self, env_name, wizard):
             assert "playmode:debug" not in env.env._options
 
 
+@pytest.mark.parametrize("env_name", [e for e in get_nethack_env_ids() if "Score" in e])
+class TestBasicGymEnv:
+    def test_inventory(self, env_name):
+        env = gym.make(
+            env_name,
+            observation_keys=(
+                "chars",
+                "inv_glyphs",
+                "inv_strs",
+                "inv_letters",
+                "inv_oclasses",
+            ),
+        )
+        obs = env.reset()
+
+        found = dict(spellbook=0, apple=0)
+        for line in obs["inv_strs"]:
+            if np.all(line == 0):
+                break
+            for key in found:
+                if key in line.tobytes().decode("utf-8"):
+                    found[key] += 1
+
+        for key, count in found.items():
+            assert key == key and count > 0
```

Comment:
Why do we expect agent to have an apple?

Reply:
This test implicitly assumes we get a monk. Monks always start with apples. :)